include/drivers/irq/riscv_plic0.h

```diff
@@ -16,7 +16,8 @@
     !defined(CONFIG_PLAT_POLARFIRE) && \
     !defined(CONFIG_PLAT_QEMU_RISCV_VIRT) && \
     !defined(CONFIG_PLAT_ROCKETCHIP_ZCU102) && \
-    !defined(CONFIG_PLAT_STAR64)
+    !defined(CONFIG_PLAT_STAR64) && \
+    !defined(CONFIG_PLAT_ARIANE)
 #error "Check if this platform suppots a PLIC."
```

Comment:
```suggestion
    !defined(CONFIG_PLAT_ARIANE)
#error "Check if this platform supports a PLIC."
```
Good moment to fix the typo too.